doc/src/manual/documentation.md

```diff
@@ -20,7 +20,8 @@ environments provide a way to access documentation directly:
   under the cursor.
 
 
-`Docs.hasdoc(module, name)::Bool` tells whether a name has a docstring.
+`Docs.hasdoc(module, name)::Bool` tells whether a name has a docstring. `Docs.undocumented_names(module; all)`
+returns the undocumented names in a module.
```

Comment:
These should also be added to the index of the reference manual somewhere.

Reply:
Is there supposed to be a `Docs/index.md` somewhere? I don't see one.

Reply:
Probably we should add a section to https://github.com/JuliaLang/julia/blob/f1f680f5605ecc471678073391f280696640207e/doc/src/base/base.md, maybe after the `## Reflection` section, like:
````
## Documentation
(See also the [documentation](@ref man-documentation) chapter.)
```@docs
Base.@doc
Docs.doc
Docs.HTML
Docs.Text
Docs.hasdoc
Docs.undocumented_names
```
````